testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -46,7 +46,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I select the addon "Legacy Module 12 x86_64"
     Then I should see the "Legacy Module 12 x86_64" selected
     # Comment following 1 line if you wish to re-enable testing with beta client tools for SLE12
-    And I deselect "SUSE Manager Client Tools Beta for SLE 12 x86_64 (BETA)" as a SUSE Manager product
+    # When I deselect "SUSE Manager Client Tools Beta for SLE 12 x86_64 (BETA)" as a SUSE Manager product
```

Comment:
We want Beta Client Tools enabled.
It will be good to check first if they are listed in the Product Wizard page.

Reply:
Right. That would need a new extra step.

Not only listed, but check, while we are at it.

I'll create a card.